includes/transformer/class-user.php

```diff
@@ -21,7 +21,15 @@ class User extends Base {
 	 * @return \Activitypub\Activity\Base_Object The ActivityPub Object
```

Comment:
This will need an update for the new return type. Any downstream uses of `to_object()` that now need wp_error handling?